ogr/services/pagure/service.py

```diff
@@ -257,14 +262,22 @@ def get_raw_request(
             ValueError, if JSON cannot be retrieved.
         """
 
+        updated_header = (
+            {**self.header, **header} if header is not None else self.header
+        )
```

Comment:
Maybe I'm missing something, but how is this different from this (except for being less readable)?
```suggestion
        headers = self.header | (header or {})
```

Reply:
Oh I see also your instruction creates a new dict.